lblod/app-digitaal-loket
# Bundle migration and add new form afwijking principes regiovorming

DL-5031

Adding new form "Afwijking principes regiovorming"
Adding "Afwijking principes regiovorming" migration

How to test : 
Run the migration and check if the administrative units from following BestuurseenheidClassificatieCodes have the form :
   - Gemente / 5ab0e9b8a3b2ca7c5e000001
   - Projectvereniging / b156b67f-c5f4-4584-9b30-4c090be02fdc
   - Dienstverlenende vereniging / d01bb1f6-2439-4e33-9c25-1fc295de2e71
   - Opdrachthoudende vereniging / cd93f147-3ece-4308-acab-5c5ada3ec63d

![Screenshot 2023-04-18 at 12-43-19 Loket voor lokale besturen](https://user-images.githubusercontent.com/38471754/232757852-d50aebef-5edb-4b4d-b757-493a7ccafdc5.png)
